src/styles.scss

```diff
@@ -1467,6 +1467,21 @@ hr {
   margin-top: 1.5rem;
 }
 
+// Surface Material primary color for use in non-Angular components/HTML
+.primary-color {
+  color: mat-color($dockstore-app-primary);
+}
+
+// To indicate success of an operation.
+.success-color {
+  color: #00bfa5;
```

Comment:
either rename class to accent-3-dark or add a comment saying accent-3-dark so that people know it came from the palette and is not random (maybe also add it as a mat-color).

Side note: This might be the first time we have a logical explanation on when to use a color that is in the palette.

Reply:
> Side note: This might be the first time we have a logical explanation on when to use a color that is in the palette.

Just going to remove the comment then.